shared/actions/chat/inbox.js

```diff
@@ -368,11 +368,24 @@ function _conversationLocalToInboxState(c: ?ChatTypes.ConversationLocal): ?Const
     }))
     .first() || {}
 
+  // Temporary hack to make team convos easier to parse in inbox view
+  const teamConvName = List(c.maxMessages || [])
+    .filter(m => m.valid && m.state === ChatTypes.LocalMessageUnboxedState.valid)
+    .map((m: any) => ({body: m.valid.messageBody, time: m.valid.serverHeader.ctime}))
+    .filter(m => [ChatTypes.CommonMessageType.metadata].includes(m.body.messageType))
+    .map((message: {time: number, body: ?ChatTypes.MessageBody}) => ({
+      title: Constants.makeTeamTitle(message.body) || '<none>',
+    }))
+    .first() || {}
+  const parts = c.info.membersType === ChatTypes.CommonConversationMembersType.team
```

Comment:
teamConvName should be figured out only if this is true, so maybe lets just make this an if statement and put that thing in side it